components/artwork/board-text.stanza

```diff
@@ -51,6 +51,7 @@ public pcb-component text (title:String, size:Double, outline-size:Double) :
   val sym = generic-symbol
   symbol = sym()
   reference-prefix = "TXT"
+  property(self.rated-temperature) = false
```

Comment:
Interesting -- does it make sense to even add this property to artwork simply because the check would be triggered otherwise? We may want to investigate how to make the check more specific. For now it's ok.

Reply:
That's right - changing the convention that geometry 'components' should be skipped in rated temperature checks.